### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by sofianelounici**
Update the presentation of Credential Digger : 

- Add a table of contents
- Simplify the process to install the tool
- Give few examples on how to launch a scan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,190 +1,145 @@
-![Logo](github_assets/Logo-CD-Mint_48.png)
 
-
-# Credential Digger
```
Comment (SlimTrabelsi) [inline README.md:4]: Why are you deleting the project description ?

File: README.md [hunk 2]
```diff
@@ -1,190 +1,145 @@
...
+```python
+from credentialdigger.cli import Client
+c = Client(dbname='MYDB', dbuser='POSTGRES_USER',
+						  dbpassword='POSTGRES_PASSWORD',
+						  dbhost='localhost', dbport=5432)
+```
+### Scan a repository
 
-## Usage (client)
+You first need to add the repository to the database. Then you can proceed to a scan :
```
Comment (marcorosa) [inline README.md:119]: `scan` already adds the repository to the database

On the contrary, what is needed before a scan is adding rules
```
c.add_rules_from_file(...)  # To be done once (before the first scan)
c.scan(...)
```

File: README.md [hunk 3]
```diff
@@ -1,190 +1,145 @@
...
-           dbhost='localhost', dbport=5432)
+c.add_repo(url='https://github.com/user/repo')
+new_discoveries = c.scan(repo_url='https://github.com/user/repo',
+                         category=None,
+                         scanner=GitScanner,
+                         models=['PathModel', 'SnippetModel'],
+                         exclude=None,
+                         force=True,
+                         generate_snippet_extractor=False,
+                         verbose=True)
```
Comment (marcorosa) [inline README.md:130]: Don't show all the parameters, but only the "most useful" ones, i.e., the repo url, the models, and verbose

File: README.md [hunk 4]
```diff
@@ -1,190 +1,145 @@
...
-   ```bash
-   cp .env.sample .env
-   vim .env  # Insert real credentials
-   ```
-
-2. Run the db and the ui using docker-compose:
-   ```bash
-   sudo docker-compose up --build
-   ```
-   Consider not to expose the db port in production.
```
[the remark this answers is not in the document]
Comment (sofianelounici) [reply README.md:66]: For me it is logic to put all the pre-requisites before all the builds/installs. Even if the docker-compose does not need the dependencies, but at least it is two sections : how you need to have before to build, and then how to install/build

File: README.md (hunk 2 above)
Comment (sofianelounici) [reply README.md:119]: Ok, I'll correct it, I added a section "Configure the regular expressions Scanner" in the installation process

File: README.md (hunk 3 above)
Comment (sofianelounici) [reply README.md:130]: Ok, I'll modify it

File: README.md [hunk 5]
```diff
@@ -1,190 +1,145 @@
...
 
-![DB Structure](github_assets/database.png)
 
 
-### Project structure
+## Requirements
 
-The project includes 3 components: a db (`sql` folder), a client
-(`credentialdigger` folder), and a user interface (`ui` folder).
+> Credential Digger supports Python 3.6 (or newer), and works only with LINUX systems.
```
[the remark this answers is not in the document]
Comment (sofianelounici) [reply README.md:26]: OK !

File: README.md [hunk 6]
```diff
@@ -1,190 +1,145 @@
...
 
-
-# Credential Digger
-
-Credential Digger is a Github scanning tool that identifies hardcoded credentials (Passwords, API Keys, Secret Keys, Tokens, personal information, etc).
-Credential Digger has a clear advantage compared to the other Github scanners in terms of False Positive reduction in the scan reports. 
-Credential Digger is using two Machine Learning Models to identify false positives, especially in Password identification:
-- Path Model: Identify the portion of code that contains fake credentials used for testing and example purposes (e.g., Unit tests).
-- Snippet Model: Identify the portion of code used to authenticate with passwords, and distinguish between real and fake passwords.
+# New Document![Logo](github_assets/Logo-CD-Mint_48.png)
```
[the remark this answers is not in the document]
Comment (sofianelounici) [reply README.md:2]: VS Code autocompletion problem, sorry !

File: README.md [hunk 7]
```diff
@@ -1,190 +1,145 @@
-![Logo](github_assets/Logo-CD-Mint_48.png)
 
-
-# Credential Digger
-
-Credential Digger is a Github scanning tool that identifies hardcoded credentials (Passwords, API Keys, Secret Keys, Tokens, personal information, etc).
-Credential Digger has a clear advantage compared to the other Github scanners in terms of False Positive reduction in the scan reports. 
-Credential Digger is using two Machine Learning Models to identify false positives, especially in Password identification:
-- Path Model: Identify the portion of code that contains fake credentials used for testing and example purposes (e.g., Unit tests).
-- Snippet Model: Identify the portion of code used to authenticate with passwords, and distinguish between real and fake passwords.
```
[the remark this answers is not in the document]
Comment (sofianelounici) [reply README.md:10]: I think we shouldn't mention the details of the models, just to mention there are models (and later to see how to install them). We can of course extend the description, but I feel introducing the 2 models here is not necessary

**Pull request merged**